python/baseline/utils.py

```diff
@@ -741,3 +743,31 @@ def f_score(overlap_count, gold_count, guess_count, f=1):
     f = (1. + beta_sq) * (precision * recall) / (beta_sq * precision + recall)
     return f
 
+@exporter
+def unzip_model(path):
+    from mead.mime_type import mime_type
```

Comment:
Need to pull mime type into bl utils for this change. Don’t want to require a dependency from bl to mead package